apps/designer/app/designer/features/props-panel/props-panel.tsx

```diff
@@ -51,11 +52,12 @@ const Property = ({
 
   return (
     <Grid
-      gap="1"
+      gap={1}
```

Comment:
its an object key in the end, so strings are actually more accurate, but I don't care ...